config/example_ros_configuration.yaml

```diff
@@ -0,0 +1,12 @@
+ueye_cam:
+  ros__parameters:
+    # Node parameters: for defaults, ueye_cam/node_parameters.hpp
+    camera_name: froody
+    frame_name: froody
+    camera_id: 0                # 0 : ANY_CAMERA
+    # Camera parameters: for defaults, ueye_cam/camera_parameters.hpp
+    auto_frame_rate: false
+    frame_rate: 2.0
```

Comment:
This is an example configuration, so this isn't critical, but I think bumping the `frame_rate` up to `10.0` or higher might be more realistic for what most people would be using. I guess it currently matches the `ids_configuration`, so maybe keeping it as it is would be fine as well.

Reply:
It's a good point - I was attempting to view the stream with `rqt_image_view` over a really really slow ssh forwarding connection. Might be worth keeping it low just so that 'it works' no matter what environment the user has.